Deployments and PR tests are failing because of missing log4j dependency

Deployments are failing with this error:

```
Caused by: org.gradle.internal.resolve.ArtifactNotFoundException: Could not find log4j-1.2.17.jar (log4j:log4j:1.2.17).
Searched in the following locations:
    https://repo.osgeo.org/repository/release/log4j/log4j/1.2.17/log4j-1.2.17.jar
```

This is probably related:
http://geoserver.org/announcements/2021/12/13/logj4-rce-statement.html

--- a/build.gradle
+++ b/build.gradle
@@ -88,6 +88,9 @@ if (project.hasProperty('env')) {
 
 allprojects {
     repositories {
+        mavenCentral().content {
+            excludeModule("javax.media", "jai_core")
+        }
         maven { url "https://repo.osgeo.org/repository/release/" }
         maven { url "https://download.osgeo.org/webdav/geotools" }
         // maven { url "http://maven.icm.edu.pl/artifactory/repo/" }
@@ -107,7 +110,6 @@ allprojects {
         maven { url "https://people.apache.org/repo/m1-ibiblio-rsync-repository/org.apache.axis2/" }
         maven { url "https://maven.geo-solutions.it" }
         mavenLocal()
-        mavenCentral()
         maven {
             url "http://nexus.onebusaway.org/content/groups/public/"
             allowInsecureProtocol = true

